lib/widgets/message_list.dart

```diff
@@ -530,6 +534,68 @@ class MarkAsReadWidget extends StatelessWidget {
   }
 }
 
+class _TypingStatusState extends State<TypingStatusWidget> with PerAccountStoreAwareStateMixin<TypingStatusWidget> {
```

Comment:
Normally we organize stateful-widget code by putting the widget first and then the state.

Reply:
Relatedly, follow the normal pattern for the relationship between widget and state classes' names: widget Foo has state _FooState.